### PR TITLE
Alternate fix for does

### PR DESCRIPTION
**Pull request by elijahbenizzy**
[Short description explaining the high-level reason for the pull request]

## Checklist

- [ ] PR has an informative and human-readable title (this will be pulled into the release notes)
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code passed the pre-commit check & code is left cleaner/nicer than when first encountered.
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future TODOs are captured in comments
- [ ] Project documentation has been updated if adding/changing functionality.

### CODE REVIEW

File: hamilton/function_modifiers.py [hunk 1]
```diff
@@ -744,7 +730,8 @@ def generate_node(self, fn: Callable, config) -> node.Node:
         and the same parameters/types as the original function.
         """
 
-        def replacing_function(__fn=fn, **kwargs):
+        # @functools.wraps(fn)
```
Comment (skrawcz) [inline hamilton/function_modifiers.py:733]: delete this

Comment (skrawcz) [reply hamilton/function_modifiers.py:733]:
```suggestion
```

**Comment by elijahbenizzy** [comment]
When we rebase let's squash and rebase cause this also has the other fix

**Comment by elijahbenizzy** [comment]
Also will have to rebase on top of this... https://github.com/stitchfix/hamilton/pull/237, which is going to be a mess.

**Comment by skrawcz** [comment]
@elijahbenizzy we should add unit tests for the other decorators you changed to verify the before and after :)

**Review by skrawcz — APPROVED** [review]
Works for me.

**Pull request merged**